platform/ios/src/MGLMapView.mm

```diff
@@ -184,7 +185,7 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingState) {
 
 @interface MGLMapView () <UIGestureRecognizerDelegate,
                           GLKViewDelegate,
-                          CLLocationManagerDelegate,
+                          MGLLocationManagerDelegate,
```

Comment:
Note that the navigation SDK currently [depends on MGLMapView conforming to CLLocationManagerDelegate](https://github.com/mapbox/mapbox-navigation-ios/blob/245901d26d9b67fe2a05e2252e39744e402a1bee/MapboxNavigation/MGLMapView%2BMGLNavigationAdditions.h#L3). I’m unsure whether the navigation SDK still takes advantage of that assumption following mapbox/mapbox-navigation-ios#402.

Reply:
The navigation SDK’s redeclaration of this conformance was vestigial: mapbox/mapbox-navigation-ios#1549.